### PR TITLE
Use 'key in dict' instead of 'key in dict.keys()'

### PR DESCRIPTION
**Pull request by MartinThoma**

### CODE REVIEW

**Review by ferdnyc — COMMENTED** [review]
:+1:  Drives me nuts.

I had some suggestions, if we're going to be fixing things anyway...

(Unfortunately, because they cross deleted lines in the diff, I couldn't leave them as _actual_ suggestions that could be applied directly from the review interface.)

**Comment by ferdnyc** [comment]
And _here_ (which extended too far outside the diff to review-comment on at all)...
https://github.com/OpenShot/openshot-qt/blob/c79d13f67be31877754f593f53e70e49471ed191/src/windows/views/files_treeview.py#L186-L195

that can all be just:
```python3
f.data.update({"name": name or os.path.basename(f.data.get("path"))})
if "tags" in f.data or tags:
    f.data.update({"tags": tags})
```
No real point in those equality tests, when **any** inequality results in an assignment. The lookup to get the current value _may_ be slightly cheaper than the assignment, but it's also completely redundant whenever the equality test fails. Might as well skip the test and just always do the assignment. If we assign the same value it already held, what's the harm?

File: src/windows/file_properties.py [hunk 1]
```diff
@@ -83,10 +83,10 @@ def __init__(self, file):
         fps_float = float(self.file.data["fps"]["num"]) / float(self.file.data["fps"]["den"])
 
         tags = ""
-        if "tags" in self.file.data.keys():
+        if "tags" in self.file.data:
             tags = self.file.data["tags"]
```
[the remark this answers is not in the document]
Comment (MartinThoma) [reply src/windows/file_properties.py:87]: I love it :heart_eyes: I'm super swamped with other work at the moment, but [I want to make this part of my flake8 plugin](https://github.com/MartinThoma/flake8-simplify/issues/50)

File: src/windows/file_properties.py [hunk 2]
```diff
@@ -82,12 +82,8 @@ def __init__(self, file):
...
         fps_float = float(self.file.data["fps"]["num"]) / float(self.file.data["fps"]["den"])
 
-        tags = ""
-        if "tags" in self.file.data:
-            tags = self.file.data["tags"]
-        name = filename
-        if "name" in self.file.data:
-            name = self.file.data["name"]
+        tags = self.file.data.get("tags", "")
+        name = file.data.get("name", filename)
```
Comment (ferdnyc) [inline src/windows/file_properties.py:86]: You dropped the 'self' there:
```suggestion
        name = self.file.data.get("name", filename)
```
(Or maybe I did, in my suggestion. Certainly possible.)

**Review by ferdnyc — CHANGES_REQUESTED** [review]
LGTM! One minor typo to fix, and then I'm good to merge. Thanks!

File: src/windows/file_properties.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ferdnyc) [reply src/windows/file_properties.py]: That would be huge, code like this is _rampant_ throughout basically all Python codebases.

File: src/windows/file_properties.py (hunk 2 above)
Comment (ferdnyc) [reply src/windows/file_properties.py:86]: Actually, I'm just gonna apply that (now that I've remembered I can do that), and then get this merged. Thanks again, @MartinThoma !

**Review by ferdnyc — APPROVED** [review]

**Pull request merged**